### PR TITLE
Add commit-based performance testing

### PR DESCRIPTION
**Pull request by adtac**

### CODE REVIEW

File: data/projects.js [hunk 1]
```diff
@@ -503,5 +503,19 @@ coala should support generating metrics for your code.",
 		"tags" : ["Bears", "cib"],
 		"markdown": "improve_installation.md",
 		"collaborating_projects" : ["coala"]
-	}
+	},
+    {
```
Comment (sils) [inline data/projects.js]: indentation is off

File: data/projects.js [hunk 2]
```diff
@@ -503,5 +503,19 @@ coala should support generating metrics for your code.",
...
+        and automatically identify and classify performance drops and gains.",
+        "requirements": [
+            "The participant should have some prior experience with investigating \
+            the performance of software."
+        ],
+        "difficulty": "medium",
+        "issues": [],
+        "intiatives": ["Paper"],
+        "tags": ["Research", "coala"],
+        "markdown": "commit_based_perftest.md"
```
Comment (sils) [inline data/projects.js]: can you add the in progress status?

File: data/projects.js (hunk 1 above)
Comment (adtac) [inline data/projects.js]: Why are we even using tabs here? o.O

File: data/projects.js (hunk 2 above)
Comment (sils) [inline data/projects.js]: just do `"in_progress": true`

File: data/projects.js (hunk 1 above)
Comment (sils) [inline data/projects.js]: because nobody added a coafile :P

**Pull request merged**